spec/serializers/api/applications_csv_serializer_spec.rb

```diff
@@ -21,7 +22,7 @@
     let(:first_row) { rows.first.to_hash.symbolize_keys }
 
     it { expect(rows.count).to eq(applications.count) }
-    it { expect(first_row.values).to all(be_present) }
+    it { expect(first_row.except(:ineligible_for_funding_reason).values).to all(be_present) }
```

Comment:
why did we need to exclude this?

Reply:
@cwrw when there is no reason this column is nil, so all values are not set. I could set it to something maybe or change the entire expect line

Reply:
Fixed